core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -253,11 +253,37 @@ export class CustomizeInteractionModalComponent
     this.isinteractionOpen = true;
   }
 
+  ButtonEnabledOnEndExploration(): boolean {
```

Comment:
function names should start with a small cased letter.

Reply:
done